### ISSUE TITLE
Speaker Consistency Loss is Missing 

### ISSUE THREAD

**Issue by talipturkmen** [tuanh123789/Train_Hifigan_XTTS#16]
Hi, 
Thank you for your work. I've reviewed your code and I notice you are missing speaker consistency loss. I think in multispeaker training it would improve the voice similarity. You can check the paper.
https://arxiv.org/pdf/2406.04904

**Comment by tuanh123789** [comment]
Yes, after the author release the paper i have implemented the speaker consistency loss. I'll update it later

**Comment by Delev1n** [comment]
Regarding this issue, is SCL added? Because I can't seem to find it in the code
